src/middleware/index.js

```diff
@@ -183,6 +186,20 @@ export let setupSocketIOAuthentication = function(app, options = {}) {
         }).catch(errorHandler);
       }
     });
+
+    socket.on('logout', function() {
```

Comment:
Ah ok I see why there is two events. But you can also send a callback from the client `socket.on('logout', callback => callback())` so you don't need two different events (that's what we're doing for calling service methods.